src/Services/ProductListDisplayService.php

```diff
@@ -0,0 +1,29 @@
+
+<?php
+
+require_once 'src/Entities/Product.php';
+
+class ProductListDisplayService
+{
+
+    public static function displayCategoryTitle(Product $productsInfo): string
+    {
+        return '<header class="container mx-auto md:w-2/3 md:mt-10 py-16 px-8 bg-slate-200 rounded">
+        <h1 class="text-5xl mb-2">Category: ' . $productsInfo->getCategoryName() . '</h1>
+        <p>For more information about any of the below products, click on the more button.</p>
+        </header>';
+    }
+
+    public static function displayProductList(Product $productsInfo): string
```

Comment:
not a great method name, this doesnt actually display a list, it just displays a single product